### DISCUSSION TITLE
Can't make it to work along with homeassistant supervised

### DISCUSSION THREAD

**Discussion by m100100** [RaspAP/raspap-webgui#831] [Q&A] [answered]
Hi,
sorry but i'm newbie, can't get raspap to work along with homeassistant supervised on the same pi. after installing the HA then i installed the Raspap, i'm able to see the wifi "raspi-webgui" but not able to connect to it. kindly advice how can make it work.
Thanks
Smile

**Comment by billz** [comment, marked as answer]
Home Assistant Supervised installs NetworkManager, which assumes dhcp duties (among other things). This conflicts with RaspAP's core networking packages and explains why your clients are unable to connect. The workaround is to install HA on another device, in a VM, or run Home Assistant Core via a Docker container.

You could also try RaspAP's Docker container (currently in Beta) https://github.com/RaspAP/raspap-docker

**Reply by m100100** [reply]
thanks alot, i also confirm the home assistant core was running just fine, i would also try the alternatives. thanks alot.

**Reply by billz** [reply]
This user created a Home Assistant add-on for RaspAP. Appears to work well. 
https://www.youtube.com/watch?v=Qx7OpaIudJQ
(video is in French)
